### ISSUE TITLE
Display Child comments of Top-Level Comments (only 1 level down, not subchildren)

### ISSUE THREAD

**Issue by OREllis** [TeamNewPipe/NewPipe#10766]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

This might be addressed under the "Read YouTube Comments" section of the FAQ, but it's not clear so ignore / mark as not following the template as needed.

On the Andrioid / Smart TV / Games Console version of the YouTube application it allows users to show child comments of a specific parent comment by selecting it, showing a list of comments that replied to it in a scrollable popup list.

Is it possible to do something similar (e.g. list child comments of top-level comment, not subchildren or sub-subchildren) using the YouTube API? 

### Why do you want this feature?

Closer functionality to that of the official YouTube application, shows discussions around specific comments on a basic level which can be expanded upon if need be.

### Additional information

E.g. Impl (not familiar with Google API specifics so if this won't work, ignore):

1. Retrieve comment ID on selction by user
2. Use youtube API to retrieve child comments by parent ID
3. show in scollable list popup, or modify comments section of the UI to show child comments in a collapsable list attached to the parent comment

**Comment by OREllis** [comment]
Can't really give a screengrab, so here's a recording showing what I mean

https://github.com/TeamNewPipe/NewPipe/assets/33124325/51b12e49-4956-4cf7-86bc-60fdb9b07d79

**Comment by OREllis** [comment]
Sorry for the shakey hand but was holding it one-handed. The idea was that, say the user long touches like how they currently do for copying comments, it opens a popup that shows a scollable list of replies to that comment specifically.

**Comment by OREllis** [comment]
To clarify, I was meaning that this should be implemented in the Mobile app version, but if it's implemented in the Android TV version that'd be good too :)

**Comment by AudricV** [comment]
Your issue is likely a duplicate of #2277. Could you check if that's the case?

**Comment by OREllis** [comment]
> Your issue is likely a duplicate of #2277. Could you check if that's the case?

Yeah, looked through it and it looks like a dupe. Thanks for catching that.

Then again, opusforlife2 originally thought this was for Android TV, could this be closed and reopened but for Andriod TV specifically?
